Core/LLVM/llvm-3.4/include/llvm/IR/IntrinsicsLunarGLASSTop.td

```diff
@@ -148,6 +148,7 @@ let TargetPrefix = "gla" in {
   def int_gla_smulExtended : Intrinsic<[llvm_anyint_ty, llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
   def int_gla_uFma : Intrinsic<[llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
   def int_gla_sFma : Intrinsic<[llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
+  def int_gla_ibMix : Intrinsic<[llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;

```

Comment:
I was just following prior art of fbMix. Should that be changed to LLVM
select as well?

Thanks,
Greg


Greg Fischer

Project Manager and Senior Engineer

LunarG, Inc. - 3D Driver Innovations

greg@lunarg.com <alan@lunarg.com>

On Tue, Apr 12, 2016 at 5:46 PM, John Kessenich <notifications@github.com>
wrote:

> In Core/LLVM/llvm-3.4/include/llvm/IR/IntrinsicsLunarGLASSTop.td
> <https://github.com/LunarG/LunarGLASS/pull/12#discussion_r59473386>:
>
> > @@ -148,6 +148,7 @@ let TargetPrefix = "gla" in {
> >    def int_gla_smulExtended : Intrinsic<[llvm_anyint_ty, llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
> >    def int_gla_uFma : Intrinsic<[llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
> >    def int_gla_sFma : Intrinsic<[llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
> > +  def int_gla_ibMix : Intrinsic<[llvm_anyint_ty], [llvm_anyint_ty, llvm_anyint_ty, llvm_anyint_ty], [IntrNoMem]>;
>
> This should not be added, as it is LLVM native as select.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/LunarG/LunarGLASS/pull/12/files/6e0efb0bbc7f140ec036c0a48411a338f700681b#r59473386>
>